lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -151,7 +151,8 @@ describe('PerfCategoryRenderer', () => {
     const passedSection = categoryDOM.querySelector('.lh-category > .lh-clump--passed');
 
     const passedAudits = category.auditRefs.filter(audit =>
-        audit.group && audit.group !== 'metrics' && Util.showAsPassed(audit.result));
+      (audit.group === 'load-opportunities' || audit.group === 'diagnostics')
+      && audit.group && Util.showAsPassed(audit.result));
```

Comment:
The `&& audit.group` seems redundant here afaict.
```suggestion
(audit.group === 'load-opportunities' || audit.group === 'diagnostics')
      && Util.showAsPassed(audit.result));
```